packages/@aws-cdk/aws-appmesh/test/test.mesh.ts

```diff
@@ -313,6 +314,127 @@ export = {
       }),
     );
 
+    test.done();
+  },
+  'Can grant an identity all read permissions for a given mesh'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const mesh = new appmesh.Mesh(stack, 'mesh', {
+      meshName: 'test-mesh',
+    });
+
+    // WHEN
+    const user = new iam.User(stack, 'test');
+    mesh.grantRead(user);
+
+    // THEN
+    expect(stack).to(haveResourceLike('AWS::IAM::Policy', {
+      PolicyDocument: {
+        Statement: [
+          {
+            Action: [
+              'appmesh:DescribeMesh',
+              'appmesh:ListMeshes',
+              'appmesh:DescribeVirtualService',
+              'appmesh:ListVirtualServices',
+              'appmesh:DescribeVirtualRouter',
+              'appmesh:ListVirtualRouter',
+              'appmesh:DescribeRoute',
+              'appmesh:ListRoutes',
+              'appmesh:DescribeVirtualNode',
+              'appmesh:ListVirtualNodes',
+              'appmesh:DescribeVirtualGateway',
+              'appmesh:ListVirtualGateways',
+              'appmesh:DescribeGatewayRoute',
+              'appmesh:ListGatewayRoutes',
+            ],
+            Effect: 'Allow',
+            Resource: [
+              {
+                Ref: 'meshACDFE68E',
+              },
+              {
+                'Fn::Join': [
+                  '',
+                  [
+                    {
+                      Ref: 'meshACDFE68E',
+                    },
+                    '/*',
+                  ],
+                ],
+              },
```

Comment:
If you could indent this to save some lines:

```suggestion
              {
                'Fn::Join': ['', [
                  { Ref: 'meshACDFE68E' },
                  '/*',
                ]],
              },
```

Same comment for the other tests.